src/SourceBuild/content/prep.sh

```diff
@@ -14,18 +14,19 @@ usage() {
     echo "  --no-bootstrap              Don't replace portable packages in the download source-built artifacts"
     echo "  --no-prebuilts              Exclude the download of the prebuilts archive."
     echo "  --no-sdk                    Exclude the download of the .NET SDK."
-    echo "  --runtime-source-feed       URL of a remote server or a local directory, from which SDKs and"
-    echo "                              runtimes can be downloaded"
-    echo "  --runtime-source-feed-key   Key for accessing the above server, if necessary"
+    echo "  --source-repository <url>   Ehen not building from a git repo, specify URL of the source"
+    echo "                              which is used for source lookup during debugging"
```

Comment:
@MichaelSimons can you please validate this sentence?

Reply:
Source Link repository URL. Required when building from tar ball.